test/unit/firestore.js

```diff
@@ -186,4 +186,25 @@ describe('MockFirestore', function () {
       db.flush();
     });
   });
+
+  describe('getAll', function() {
+    it('', function() {
+      var doc1 = db.doc('doc1');
+      var doc2 = db.doc('doc2');
+      var doc3 = db.doc('doc3');
+
+      db.autoFlush();
```

Comment:
call `db.flush()` instead after setting the values

Reply:
Updated.